src/test/java/org/broadinstitute/hellbender/engine/FeatureDataSourceUnitTest.java

```diff
@@ -666,4 +646,7 @@ public void testHandleCachingOfEmptyRegion() {
     /*********************************************************
      * End of direct testing on the FeatureCache inner class
      *********************************************************/
+
+    @Test
+    public void testGenomicsDBFeatureInput(){}
```

Comment:
Deleted the empty test -- I'll write new tests for the new functionality in `FeatureInput`